gluten-flink/runtime/src/main/java/org/apache/gluten/table/runtime/operators/GlutenSingleInputOperator.java

```diff
@@ -50,19 +49,17 @@
 import org.slf4j.LoggerFactory;
 
 import java.util.List;
-import java.util.concurrent.BlockingQueue;
-import java.util.concurrent.LinkedBlockingQueue;
 
 /** Calculate operator in gluten, which will call Velox to run. */
 public class GlutenSingleInputOperator extends TableStreamOperator<RowData>
         implements OneInputStreamOperator<RowData, RowData>, GlutenOperator {
 
     private static final Logger LOG = LoggerFactory.getLogger(GlutenSingleInputOperator.class);
 
-    private final PlanNode glutenPlan;
+    private final Velox4JBean<PlanNode> glutenPlan;
     private final String id;
-    private final RowType inputType;
-    private final RowType outputType;
+    private final Velox4JBean<RowType> inputType;
+    private final Velox4JBean<RowType> outputType;
```

Comment:
If outputType changes to Map<String, RowType>, does it work?

Reply:
This will work: `Map<String, Velox4JBean<RowType>>`